### DISCUSSION TITLE
403 Error when searching with cross tenant Synced accounts.

### DISCUSSION THREAD

**Discussion by JordanMastel** [microsoft-search/pnp-modern-search#3145] [Q&A]
**Version used**
4.8

I did not see an existing issue for this or a fix within the patch notes for 4.9.

**Describe the bug**

When using Cross tenant Synced accounts, accounts that were previously guest accounts are unable to use search. I get a 403 forbidden error when they are converted to member accounts.

**To Reproduce**
Detailed steps to reproduce the behavior:
Do a search
In search results see a 403 error. 

**Expected behavior**

**Screenshots**
![image](https://github.com/microsoft-search/pnp-modern-search/assets/96920984/80811bae-f646-434f-aa58-69cb41f2cb3e)
[HTTP]:403 - Forbidden [CorrelationId]:c371c7a0-4038-3000-d368-62e2a2bd5a1a [Version]:16.0.0.23905

**Desktop (please complete the following information):**
Chrome, Edge, InCognito and InPrivate

**Additional context**
Cross tenant synchronization is a new feature released this year.

**Comment by kasperbolarsen** [comment]
Please verify if the same issue is present when using classic search in order to determining if this is a PnP issue or not
